core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -301,6 +311,15 @@ public int getValue() {
    */
   String getFingerprint(String path);
 
+  /**
+   * A logical UFS may be composed of one or more physical UFSs. This method is used to determine
```

Comment:
What is a "physical UFS"? This is related to my questions about "physical path".